src/components/widgets-settings/chart-filters/chart-filters.less

```diff
@@ -14,6 +14,7 @@
     outline: none;
   }
   input[type="range"] {
-    height: 20px;
+    height: auto;
```

Comment:
have you tested this does not impact the other widgets that already use a range input?

Reply:
Yes the other widgets are good to go in both IE and chrome.

Reply:
:+1: